aws-xray/src/awsTest/java/io/opentelemetry/contrib/aws/xray/AwsXrayRemoteSamplerIntegrationTest.java

```diff
@@ -49,29 +51,38 @@
 
   @Test
   void keepSampling() throws Exception {
-    AwsXrayRemoteSampler sampler =
+    // Initialize two samplers to try to see centralized reservoir behavior.
+    AwsXrayRemoteSampler sampler1 =
+        AwsXrayRemoteSampler.newBuilder(Resource.getDefault())
+            .setEndpoint("http://localhost:" + otelCollector.getMappedPort(2000))
+            .setPollingInterval(Duration.ofSeconds(5))
+            .build();
+
+    AwsXrayRemoteSampler sampler2 =
         AwsXrayRemoteSampler.newBuilder(Resource.getDefault())
             .setEndpoint("http://localhost:" + otelCollector.getMappedPort(2000))
             .setPollingInterval(Duration.ofSeconds(5))
             .build();
 
     try {
       while (true) {
```

Comment:
Should've asked the first time around - but when does this loop exit? Or is it not intended to exit, because it's only manually runnable so the human is meant to terminate it?

Reply:
Right since it's just for running manually this doesn't exit until user kills the test